code/modules/mob/say.dm

```diff
@@ -1,17 +1,25 @@
 //Speech verbs.
 // the _keybind verbs uses "as text" versus "as text|null" to force a popup when pressed by a keybind.
+var/global/message_length = 0
```

Comment:
Global vars here are made through `GLOBAL_VAR(varname, value)`, though like I said above, isnt really needed to be a global.

Reply:
thank you, changing it now! Though it will have to wait a little more as I'm trying to add a global preference function to mute all vocals